ebpfapi/ebpfapi.vcxproj

```diff
@@ -35,31 +13,31 @@
     <ProjectName>EbpfApi</ProjectName>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'" Label="Configuration">
```

Comment:
I believe these were replicated so that they could be edited in VS whereas the changed version probably cannot.

Reply:
My two cents: .vcxproj files should be optimized for human readability and human authoring, i.e., human maintenance, while remaining readable by VS.

Example: https://github.com/microsoft/xdp-for-windows/blob/main/src/xdplwf/xdplwf.vcxproj

With this style of .vcxproj factoring, making repo-wide, directory-wide, or architecture-specific/config-specific changes requires changing only one .props file, rather than editing multiple configurations in the VS UI, which in turn are very difficult to review.

Reply:
Another example: a simple project has a simple .vcxproj file:

https://github.com/microsoft/xdp-for-windows/blob/main/test/xskbench/xskbench.vcxproj

Adding arm64 support for this project required zero changes to the .vcxproj file, which seems like an ideal outcome for a project without any architecture-specific logic.

Reply:
My view is that we should meet developers where they are at.  E.g., we don't force developers to use vi or emacs or VS or VS Code, we let them use whatever editor they like.  "optimized for" is different from "only support".  Currently the changes in this PR break some workflows and force developers to use a text editor instead.  That's what I don't like.

Reply:
I think the file format should be optimized for those who are actually expected to make the changes, and not just because the VS UI does it some way. That produces a lot of duplication and makes it hard for the actual maintainers to do their jobs. I'm in favor of these type of changes.

Reply:
PR is has been trimmed down considerably. Hope this helps.